components/Episode/EpisodeFunctions.js

```diff
@@ -23,3 +23,64 @@ export function getFormattedDate(date) {
     return false;
   }
 }
+
+// https://stackoverflow.com/questions/10123953/how-to-sort-an-object-array-by-date-property/26759127#26759127
+
+export function sortFirstToLast(episodes) {
+  return episodes.slice().sort(function (a, b) {
+    return (
+      new Date(a.veröffentlichungsdatum) - new Date(b.veröffentlichungsdatum)
+    );
+  });
+}
+
+export function sortLastToFirst(episodes) {
+  return episodes.slice().sort(function (a, b) {
+    return (
+      new Date(b.veröffentlichungsdatum) - new Date(a.veröffentlichungsdatum)
+    );
+  });
+}
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+export function getRandomIntInclusive(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1) + min); // The maximum is inclusive and the minimum is inclusive
+}
+
+export function getMostRecentEpisode(episodes) {
+  return episodes
+    .slice()
+    .sort(function (a, b) {
+      return (
+        new Date(a.veröffentlichungsdatum) - new Date(b.veröffentlichungsdatum)
+      );
+    })
+    .findLast((element) => {
+      return element;
+    });
```

Comment:
This is a very unusual way of retrieving the last element of an array, you can use: `myArray.at(-1)`